.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
```

Comment:
@chukwuemekachm good job here. I am wondering though, is there a reason why the config for circleci is in a directory instead of on the root of the repo like the others? If it is not a requirement of the integration, I believe having it on the root will make it consistent with the rest.

Reply:
Nice observation @Veraclins. However, circle-ci version 2 requires a `.circle-ci` folder which contains lots of settings and config files.

Reply:
Yes, Travis-ci and circle-ci basically do the same thing. I decided to add circle-ci due to the issues we had during our first phase with Travis-ci taking too long to build because it had to queue the build process thereby holding of merges until the build status has been verified. Travis-ci is the default test builder and coverage report submitter, but when it takes too long to build a branch we can fall back to the results from circle-ci and easily make decisions from that.

Reply:
I still think we can use one of them. If Travis is not reliable, we can forget it and use circle-ci. Unless there are some things that circle-ci cannot do.